### PR TITLE
Added code to quick copy all env vars

### LINKED ISSUE #6114: [DOCS] Quick copy all required env vars
### Description

The variables documentation has a nice table of all the variables and what they mean, but you must copy/paste individually into your project.  The idea for this issue would be to create a button on that page to copy the required variable to be able to quickly post them to your local .env file.

docs page: https://linkfree.io/docs/environments/environment-variables

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by PratikDey**
Closes #6114 

## Fixes Issue
Added code as per the issue suggestion

## Changes proposed
Added Clipboard copy to environment docs to quick copy all environment variables

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7069"></a>

### CODE REVIEW

File: pages/docs/environments/environment-variables.mdx [hunk 1]
```diff
@@ -15,6 +16,20 @@ Environment variables are needed in the `.env` file to allow the application to
...
+```
+LINKFREE_MONGO_CONNECTION_STRING="mongodb://localhost:27017/linkfree"
+NEXT_PUBLIC_GA_MEASUREMENT_ID=""
+NEXT_PUBLIC_BASE_URL="http://localhost:3000"
+NODE_ENV="development"
+
+GITHUB_ID=""
+GITHUB_SECRET=""
+NEXTAUTH_SECRET=""
+NEXTAUTH_URL="http://localhost:3000"
```
Comment (eddiejaoude) [inline pages/docs/environments/environment-variables.mdx:29]: This gets converted to...

```
LINKFREE_MONGO_CONNECTION_STRING=&quot;mongodb://localhost:27017/linkfree&quot;
NEXT_PUBLIC_GA_MEASUREMENT_ID=&quot;&quot;
NEXT_PUBLIC_BASE_URL=&quot;http://localhost:3000&quot;
NODE_ENV=&quot;development&quot;

GITHUB_ID=&quot;&quot;
GITHUB_SECRET=&quot;&quot;
NEXTAUTH_SECRET=&quot;&quot;
NEXTAUTH_URL=&quot;http://localhost:3000&quot;
```

**Review by eddiejaoude — DISMISSED** [review]
Thank you for your contribution. Please test the changes, I think there is an encoding issue

**Comment by PratikDey** [comment]
Hi, this PR is ready for review

**Review by ChinmayMhatre — APPROVED** [review]
Tested on gitpod. Works on my side.

**Comment by eddiejaoude** [comment]
Great collab 👍

**Pull request merged**